meshmode/array_context.py

```diff
@@ -702,13 +670,13 @@ def thaw(actx, ary):
 
     :param ary: a :meth:`~ArrayContext.freeze`\ ed :class:`ArrayContainer`.
     """
+    return _thaw_inner(ary, actx)
     if not is_array_container(ary):
         raise TypeError(
                 f"cannot thaw arrays of type {type(ary).__name__}; "
                 "try calling 'ArrayContext.thaw' directly")
 
-    return _map_array_container_with_context(
-            actx.thaw, ary, actx=actx, recursive=True)
+    return _map_array_container(actx.thaw, ary, recursive=True)
```

Comment:
Delete?

Reply:
This seems to have clipped weirdly, but there's a return on the line *just* above already.

Reply:
959af1f